client/src/pages/mainPage/components/home/Home.js

```diff
@@ -0,0 +1,165 @@
+	},
+	{
+		Name: "Find the Colour",
+		Image: IMG,
+		Description:
+			"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+		link: "https://sample.com",
+		id: "3",
+	},
+	{
+		Name: "Vacation Destination",
+		Image: IMG,
+		Description:
+			"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+		link: "https://sample.com",
+		id: "4",
+	},
+	{
+		Name: "Favorite Movie Star",
+		Image: IMG,
+		Description:
+			"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+		link: "https://sample.com",
+		id: "5",
+	},
+	{
+		Name: "Best Friend ",
+		Image: IMG,
+		Description:
+			"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+		link: "https://sample.com",
+		id: "6",
+	},
+	{
+		Name: "Emoji",
+		Image: IMG,
+		Description:
+			"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+		link: "https://sample.com",
+		id: "7",
+	},
+	{
+		Name: "Weird Talent",
+		Image: IMG,
+		Description:
+			"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+		link: "https://sample.com",
+		id: "8",
+	},
+	{
+		Name: "Bucket List",
+		Image: IMG,
+		Description:
+			"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+		link: "https://sample.com",
+		id: "9",
+	},
+	{
+		Name: "Fav Youtube Video",
+		Image: IMG,
+		Description:
+			"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+		link: "https://sample.com",
+		id: "10",
+	},
+];
+
+export function Home() {
+	// const [energizers, setEnergizers] = useState();
```

Comment:
You could add `[]` as the default argument here so you don't need to check if energisers is null

Reply:
Thank you.
I'll be deleting this as soon as we're able to fetch from the database.